src/merge.js

```diff
@@ -0,0 +1,3 @@
+import merge from 'lodash.merge';
+
+export default merge;
```

Comment:
I don't know about babel. but I know lodash will use Object.assign if available, if not it's own implementation. native Object.assign might be faster? I don't know, nor do i think it can actually have any impact what so ever. I use lodash everywhere, and it has never let me down. But babel doesn't seem bad either. I'm alright with both approaches :P